lib/ss_light.go

```diff
@@ -384,5 +392,24 @@ func (l *LightClient) Start(
 	if err != nil {
 		log.Warn("Failed updating metadata after download Err: %s", err.Error())
 	}
-	return "Download complete", nil
+	if !stat {
+		return "Download complete", nil
+	}
+
+	connectedPeerIds := lite.Host.Network().Peers()
```

Comment:
This variable is not required. You can directly use lite.Host.Network().Peers() in the for loop